packages/contracts/mud.config.ts

```diff
@@ -166,6 +166,7 @@ export default mudConfig({
         startFrom: "uint32", // current round start block timestamp
         finishedBoard: "uint8",
         globalRandomNumber: "uint256",
+        single: "bool",
```

Comment:
move `single` to the top of `globalRandomNumber`

Reply:
why ? Is there a better way to write this

Reply:
for putting small avariables together in a slot.